naff/client/const.py

```diff
@@ -96,6 +98,7 @@ def get_logger() -> logging.Logger:
 
 default_locale = "english_us"
 kwarg_spam = False
+tag_as_mention = True
```

Comment:
Why is this in const?
It should:
A. Be a parameter on contains_mention
B. Be False by default.

Reply:
Well... I was tired yesterday

I'll change it later

Reply:
changed with [03e2f64](https://github.com/NAFTeam/NAFF/pull/700/commits/03e2f64f2de76eca3749fa40cc8dad285ed7789b)